prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -109,63 +117,86 @@ func handlePullRequest(pc plugins.Agent, pre github.PullRequestEvent) error {
 	return handle(
 		pc.GitHubClient,
 		oc, pc.Logger,
-		pc.PluginConfig.Blunderbuss.ReviewerCount,
-		pc.PluginConfig.Blunderbuss.FileWeightCount,
-		pc.PluginConfig.Blunderbuss.MaxReviewerCount,
-		pc.PluginConfig.Blunderbuss.ExcludeApprovers,
-		&pre,
+		pc.PluginConfig.Blunderbuss,
+		&pre.Repo,
+		&pre.PullRequest,
+	)
+}
+
+func handleGenericCommentEvent(pc plugins.Agent, ce github.GenericCommentEvent) error {
+	if ce.Action != github.GenericCommentActionCreated || !ce.IsPR || ce.IssueState == "closed" {
+		return nil
+	}
+
+	pr, err := pc.GitHubClient.GetPullRequest(ce.Repo.Owner.Login, ce.Repo.Name, ce.Number)
```

Comment:
Feels like we should do this in the `handle`?

Reply:
We only need it when handling a comment since it's not available in `github.GenericCommentEvent`. `github.PullRequestEvent` already contains the `github.PullRequest` struct so it's not needed it in that case.